src/explore-education-statistics-admin/src/pages/release/pre-release/components/PreReleaseUserAccessForm.tsx

```diff
@@ -81,28 +102,43 @@ const PreReleaseUserAccessForm = ({
         <Formik<FormValues>
           enableReinitialize
           initialValues={{
-            email: '',
+            emails: '',
           }}
           validationSchema={Yup.object<FormValues>({
-            email: Yup.string()
-              .required('Enter an email address')
-              .email('Enter a valid @education.gov.uk email address')
+            emails: Yup.string()
+              .trim()
+              .required('Please enter 1 or more email addresses')
+              .test({
+                name: 'number of lines',
+                message: `Please enter between 1 and ${inviteLimit} lines`,
```

Comment:
No need to use 'please'. Would also reword this to 'Enter between 1 and 50 lines of email addresses'

Reply:
Updated to `Enter between 1 and 50 lines of email addresses`. Thanks.